demo/basic/tabs.component.ts

```diff
@@ -58,11 +63,34 @@ import { Component } from '@angular/core';
       </div>
 
     </div>
-  `
+  `,
+  styles: [`
```

Comment:
Styles should be inline.

Reply:
@amcdnl inline how? I'm not sure I'm following

Reply:
They should live in a separate file vs inline in the component.

Reply:
thanks, will do